sdk/test-utils/test-utils/src/chai.ts

```diff
@@ -0,0 +1,9 @@
+import chai from "chai";
+import { chaiAzure } from "./chaiAzure";
+import chaiAsPromised from "chai-as-promised";
+chai.use(chaiAzure);
+chai.use(chaiAsPromised);
+const assert = chai.assert;
+
+export type { chaiAsPromised };
```

Comment:
I have no idea what I'm doing here 🤣 but hopefully someone can let me know if there's a better way

Reply:
What this does is export a type symbol from this module called `chaiAsPromised` that is the type of the module's default export. I think as long as the `import "chai-as-promised"` is referenced in the module graph, it should work without having to export this symbol. If it doesn't, then something weird must be going on.

You may also just be able to do `/// <reference types="chai-as-promised" />` and that will have the same effect.

Reply:
If you noticed I _only_ had to do this for chai-as-promised, not for chaiAzure (where we define the types within the ts file). I wonder if it's because types for chai-as-promised are pulled in from the separate @types/chai-as-promised package? I'll try the `reference` though as well

Reply:
No I lied, I do need to export both from chai, but not from index for whatever reason...